web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1663,116 +1663,116 @@ export const formatters: Formatters = {
   [eventCodes.ACCESS_LIST_CREATE]: {
     type: 'access_list.create',
     desc: 'Access list created',
-    format: ({ name, updated_by }) =>
-      `User [${updated_by}] created access list [${name}]`,
+    format: ({ access_list_title, updated_by }) =>
+      `User [${updated_by}] created access list [${access_list_title}]`,
```

Comment:
`access_list_title` field will only be available for the newer audit events. 
Don't we need to fallback to `name` if the title field does not exist?

Reply:
Something like this?